wdp/database/connector_wdp.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+
+class Database:
+    def __init__(self, database_name="wdp_database.db"):
+        # create connection to database
+        self.connection = sqlite3.connect(database_name)
+        self.cursor = self.connection.cursor()
+
+    def __del__(self):
+        # close connection to database after function usage
+        self.connection.close()
+
+    def insert_sql(self, table, *values):
+        # insert values into table
+        self.cursor.execute(f"INSERT INTO {table} VALUES ({','.join(['?' for _ in values])})", values)
+        self.connection.commit()
+        print(f'Row inserted to {table} ')
```

Comment:
Use a logger, don't print anything to the console.
I suggest replacing all `print` statements with `logger.debug` and above the class declaration just simply put `logger = logging.getLogger(__name__)` (and don't forget to import `logging` first).